lib/forms/submission_statuses/formatter.rb

```diff
@@ -6,7 +6,7 @@ class Formatter
       def format_data(dataset)
         return [] unless dataset.submissions?
 
-        results = merge_records(dataset.submissions, dataset.statuses)
+        results = merge_records(dataset.submissions, dataset.intake_statuses)
```

Comment:
Refactored by using helper methods to interact with the dataset, instead of directly.  This should help address the issue, but it's more of a workaround than a full resolution.  I have some ideas to improve this, but I think this refactoring is good enough for now.